index.html

```diff
@@ -9,79 +9,101 @@
+        </div>
+        <div class="row skills">
+            MARKETING + BRANDING + WEB DESIGN + WORD PRESS
         </div>
     </section>
-    <section class="container">
+    <section class="container bg-light-white">
         <div class="row">
-            <h2 class="section-title">Our Latest Featured Projects</h2>
-            <p class="section-description">Who are in extremely love with eco friendly system.</p>
+            <h2 class="section-title">Best Services</h2>
+            <p class="section-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
         </div>
         <div class="row">
-            PORTFOLIO FILTER + GALLERY
+            SERVICES LIST
         </div>
     </section>
     <section class="container bg-light-grey">
         <div class="row">
-            <h2 class="section-title">Client’s Feedback About Me</h2>
-            <p class="section-description">It is very easy to start smoking but it is an uphill task to quit it. Ask any chain smoker or even a person.</p>
+            <h2 class="section-title">Portfolio Grid</h2>
+            <div> 
+                ALL + DESIGN THINKING + BRANDING + PHOTOGRAPHY + DIGITAL MARKETING + TECHNOLOGY
+            </div>
         </div>
         <div class="row">
-            TESTIMONIALS LIST
+            PORTFOLIO LIST
         </div>
     </section>
-    <section class="container">
+    <section class="container bg-light-white">
         <div class="row">
-            <h2 class="section-title">Choose Your Plan</h2>
-            <p class="section-description">When someone does something that they know that they shouldn’t do, did they.</p>
+            <h2 class="section-title">Customer Testimonials</h2>
+            <p class="section-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent diam lacus, dapibus sed imperdiet.</p>
         </div>
         <div class="row">
-            PRICING LIST
+            CUSTOMER LIST
         </div>
     </section>
     <section class="container bg-light-grey">
         <div class="row">
-            <h2 class="section-title">Latest Posts From Our Blog</h2>
-            <p class="section-description">You may be a skillful, effective employer but if you don’t trust your personnel and the opposite, then the chances of improving and expanding the business</p>
+            <h2 class="section-title">Let's Talk?</h2>
+            <p class="section-description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec congue lectus augue. Sed pharetra ornare vulputate.</p>
         </div>
-        <div class="row">
-            BLOG POSTS LIST
+        <div class="row progress-bar">
+            APLICATION PROGRESS
+        </div>
+        <div class="row form">
+            FORM
         </div>
     </section>
-    <section class="container bg-light-blue">
+    <section class="container bg-dark">
```

Comment:
Gal sita konteineri prie footerio priklijuot? viena klase maziau butu

Reply:
Footerio paddingas yra kitoks negu kitų konteinerių ir jis eina per visus vidinius puslapius, tai dėl to dariau atskirą klasę footerio konteineriui. Man atrodė toks geriausias sprendimas, turbūt dar kažkaip galima, bet vistiek kažkas atskiro bus

Reply:
as gal blogai issireiskiau...  man pasirode is projekto, kad taviskis "container bg-dark", priklauso footeriui :)